(1) Community Patch/Core Files/Text/CoreText_en_US.xml

```diff
@@ -5728,22 +5728,19 @@
 			<Text>Marriage between {@2_MinorCivName} and {@1_CivName}!</Text>
 		</Row>
 		<Row Tag="TXT_KEY_NOTIFICATION_MINOR_MARRIAGE_TT_1">
-			<Text>A beautiful princess of {@1_CivName} has been married off to a nobleman of {@2_MinorCivName}. With a diplomatic relationship united by marriage, {@2_MinorCivName} grants {@1_CivName} an additional Delegate in the World Congress, boosts their [ICON_CAPITAL] Capital's [ICON_GREAT_PEOPLE] Great Person rate, and increases their resting [ICON_INFLUENCE] Influence with the City-State by 200.</Text>
+			<Text>A beautiful princess of {@1_CivName} has been married off to a nobleman of {@2_MinorCivName}. With a diplomatic relationship united by marriage, your resting [ICON_INFLUENCE] Influence with {@2_MinorCivName} is increased by 75, scaling with Era, and your [ICON_CAPITAL] Capital's [ICON_GREAT_PEOPLE] Great Person rate is increased by 15%.</Text>
```

Comment:
Given that this is a notification, is it possible to add a {@3_RestingInfluence} and send in the actual increase instead of being hard-coded to "75, scaling with Era"?

Reply:
The simplest, instead of an increase, is to state the new resting influence.  Edit `CvMinorCivAI->DoBuyout()`:
```c++
	if (bMarriage)
	{
		strMessage = iCoinToss == 0 ? Localization::Lookup("TXT_KEY_NOTIFICATION_MINOR_MARRIAGE_TT_1") : Localization::Lookup("TXT_KEY_NOTIFICATION_MINOR_MARRIAGE_TT_2");
		strMessage << GET_PLAYER(eMajor).getCivilizationShortDescriptionKey();
		strMessage << GetPlayer()->getCivilizationShortDescriptionKey();

		strSummary = Localization::Lookup("TXT_KEY_NOTIFICATION_MINOR_MARRIAGE");
		strSummary << GET_PLAYER(eMajor).getCivilizationShortDescriptionKey();
		strSummary << GetPlayer()->getCivilizationShortDescriptionKey();
		strSummary << GetFriendshipAnchorWithMajor(eMajor); // new line
	}
```
and then the text would look something like:
```xml
<Text>A beautiful princess of {@1_CivName} has been married off to a nobleman of {@2_MinorCivName}. With a diplomatic relationship united by marriage, your resting [ICON_INFLUENCE] Influence with {@2_MinorCivName} is increased to {@3_RestingInfluence}, and your [ICON_CAPITAL] Capital's [ICON_GREAT_PEOPLE] Great Person rate is increased by 15%.</Text>
```